src/main/java/com/jalasoft/selenium/daniel/jauregui/movies/Customer.java

```diff
@@ -0,0 +1,77 @@
+package com.jalasoft.selenium.daniel.jauregui.movies;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+/**
+ * Customer class.
+ * @author Daniel Jauregui
+ */
+class Customer {
+    private final String name;
+    private final ArrayList<Rental> rentals = new ArrayList<Rental>();
+    /**
+     * Constructor of Customer class.
+     * @param name : Name of the customer.
+     */
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * addRental will add the movies that customer is renting.
+     * @param arg : will have a Rental object.
+     */
+    public void addRental(final Rental arg) {
+        this.rentals.add(arg);
+    }
+
+    /**
+     * getRentals.
+     * @return : Get Object with all rentals of customer.
+     */
+    public ArrayList<Rental> getRentals() {
+        return this.rentals;
+    }
+
+    /**
+     * Get the Name of Customer.
+     * @return : Will return the name of Customer.
+     */
+    public String getName() {
+        return this.name;
+    }
+
+    /**
+     * Generate the Statement Rentals of Customer.
+     * @return : Will return the summary of Customer Statement
+     */
+    public String statement() {
```

Comment:
This method can be split in 3 methods
currently is calculating totalAmount, totalFrequentRenterPoints and is printing.